File: 0-level/C-Sharp-Basics/04.ConsoleInputOutput/Circle/Circle.cs
using System;

class Circle
{
    static void Main()
    {
        Console.Write("Enter the radius of the circle: ");
        double rad = float.Parse(Console.ReadLine());
        double per = 2 * Math.PI * rad;
        double area = Math.PI * rad * rad;
        Console.WriteLine("The perimeter of the circle is: {0:F2}", per);
        Console.WriteLine("The area of the circle is: {0:F2}", area);
    }
}

File: 2-level/OOP/OOP-in-Java/src/geometry/shape/plane/Circle.java
package geometry.shape.plane;


public class Circle extends PlaneShape {
	private double radius;
	
	public Circle(double x, double y, double radius) {
		super(x, y);
		this.setRadius(radius);
	}

	public double getRadius() {
		return radius;
	}

	public void setRadius(double radius) {
		if (radius <= 0) {
			throw new IllegalArgumentException("The circle radius, cannot be zero or negative!");
		}
		
		this.radius = radius;
	}
	
	@Override
	public double getArea() {
		double area = Math.PI * this.radius * this.radius;
		
		return area;
	}

	@Override
	public double getPerimeter() {
		double perimeter = 2 * Math.PI * this.radius;
		
		return perimeter;
	}

}
